tests/scale_test/notifier.py

```diff
@@ -0,0 +1,239 @@
+
+    Note: For this batch mode, you could also specify whether to use multi-threading of not. By default, multi-threading
+        is enabled. It will send notifications asynchronously, which is much faster. Besides, for the file to be passed
+        into this command, it need to be in the following JSON schema:
+        [
+            {"bundle_uuid": XXX, "bundle_version": XXX},
+            {"bundle_uuid": XXX, "bundle_version": XXX}
+        ]
+
+"""
+import click
+import logging
+from tqdm import tqdm
+import time
+import utils
+from collections import deque
+import concurrent.futures
+import functools
+import timeit
+import pathlib
+
+
+logging.basicConfig(level=logging.INFO)
+
+
+@click.group()
+def cli():
+    pass
+
+
+@cli.group(context_settings=dict(max_content_width=120))
+@click.option('--lira_url',
+              default='https://pipelines.dev.data.humancellatlas.org/',
+              help='The url to a Lira instance.',
+              show_default=True)
+@click.option('--label',
+              default='scaling-test-{}'.format(time.strftime('%Y-%m-%d')),
+              help='The label to be added to the workflows from notifications. '
+                   'Note: the label should comply with the Cromwell\'s rules!!',
+              show_default=True)
+@click.option('--save_path',
+              default='./',
+              help='The path to the folder to save metrics of this notifier.',
+              show_default=True)
+@click.pass_context
+def notify(ctx, lira_url, label, save_path):
+    ctx.obj['lira_url'] = utils.harmonize_url(lira_url)
+    ctx.obj['label'] = label
+    ctx.obj['save_path'] = save_path
+
+
+@notify.command(short_help='Send one notification from a given bundle UUID and bundle VERSION.')
+@click.pass_context
+@click.option('--uuid',
+              help='A bundle UUID in Blue Box.')
+@click.option('--version',
+              help='A bundle VERSION matches the UUID in Blue Box')
+def once(ctx, uuid, version):
+    # Using a custom checker instead of the required=True flag of Click, so it shows the help text
+    utils.required_checker(ctx, uuid=uuid, version=version)
+
+    # Checking if the auth_token given is correct
+    auth_token = utils.auth_checker(ctx)
+
+    # Prepare arguments
+    lira_url, label = ctx.obj['lira_url'], ctx.obj['label']
+
+    # Use a probe to get the current subscription_id
+    subscription_id = utils.subscription_probe(lira_url)
+    logging.info('Using subscription_id: {}'.format(subscription_id))
+
+    # Prepare
+    notification = utils.prepare_notification(uuid, version, subscription_id, label)
+
+    # Send notifications
+    start = timeit.default_timer()
+    response = utils.send_notification(lira_url, auth_token, notification)
+    stop = timeit.default_timer()
+
+    # Output metrics
+    logging.info('Sent notification with Round-Trip-Time: {rtt}, in {total_time} seconds, status is {status}'.format(
+        rtt=response.elapsed.total_seconds(),
```

Comment:
What is the difference between the round-trip-time and the total time?

Reply:
I also changed the place to start and stop the timer, now. So rtt is the time spent on requests, while total_time is the total running time of the command. Thanks for cathing this!!